primitives/src/config.rs

```diff
@@ -71,7 +19,7 @@ pub struct RateLimit {
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct Config {
-    pub identity: String, // should not be here maybe?
+    pub identity: Option<String>, // should not be here maybe?
```

Comment:
What's the point of not having an identity?

Reply:
again, shouldn't be an option

Reply:
I think identity shouldn't be part of `Config` becuase it's not a configuration hence the option<String> to make the code compile